ingestion/src/metadata/profiler/processor/sampler/pandas/sampler.py

```diff
@@ -136,7 +136,7 @@ def get_col_row(self, data_frame, columns: Optional[List[SQALikeColumn]] = None)
                 break
         return cols, rows
 
-    def random_sample(self):
+    def random_sample(self, is_sampled: bool = False):
```

Comment:
what is that argument for?

Reply:
as we fetch the data frames ( already sampled ) this is to make sure that they don't get sampled again

Reply:
Do you mean when we parse the rows to evaluate the types?

Reply:
as we call `return_ometa_dataframes_sampled the dataframe comes sampled, having to do a random sample over it again won't be needed that's why the extra arg